vcbuild.bat

```diff
@@ -117,7 +117,7 @@ if "%i18n_arg%"=="full-icu" set configure_flags=%configure_flags% --with-intl=fu
 if "%i18n_arg%"=="small-icu" set configure_flags=%configure_flags% --with-intl=small-icu
 if "%i18n_arg%"=="intl-none" set configure_flags=%configure_flags% --with-intl=none
 if "%i18n_arg%"=="without-intl" set configure_flags=%configure_flags% --without-intl
-if "%engine%"=="chakracore" set configure_flags=%configure_flags% --without-intl --without-inspector
+if "%engine%"=="chakracore" set configure_flags=%configure_flags% --without-intl --without-inspector --without-v8-platform --without-bundled-v8
```

Comment:
> Because every time a new condition is added, you have to arrange conditions in node.gyp to make it under top level condition. I feel what we do in vcbuild.bat is simpler.

I'm worried if you rearrange (move) those conditions, it'll make future merge harder. Alternatively, you could just add "engine='v8' and" to the conditions without moving them (unless running into condition conflicts case like last time).